src/main.jl

```diff
@@ -18,11 +18,17 @@ function main(
     bump_compat_containing_equality_specifier=true,
     pr_title_prefix::String="",
     include_jll::Bool=false,
+    unsub_from_prs=false,
+    cc_user=false,
 )
     if ci_cfg isa GitHubActions
         token = GitHub.Token(github_token())
         api = GitHub.GitHubAPI(; token=token, url=hostname_for_api)
         repo, _ = @mock GitForge.get_repo(api, github_repository())
+    elseif ci_cfg isa GitLabCI
+        token = GitLab.PersonalAccessToken(gitlab_token())
+        api = GitLab.GitLabAPI(; token=token, url=hostname_for_api)
+        repo, _ = @mock GitForge.get_repo(api, gitlab_repository())
     else
```

Comment:
Maybe pull these 3 branchs out into little methods that return the `api` and the `repo`?